### ISSUE TITLE
Parsing the command line arguments

### ISSUE THREAD

**Issue by vitoyucepi** [savonet/liquidsoap#3409]
**Is your feature request related to a problem? Please describe.**
I'd like to get `argv` as a list instead of a function.
It will make it possible to determine the number of arguments.

**Describe the solution you'd like**
I believe only the first argument, which is not an option, should be treated as a script.
1. `liquidsoap 1.liq 2.liq` will run liquidsoap with the script *1.liq*.
   `argv` should be equal to `["1.liq", "2.liq"]`.
1. `./1.liq 2.liq` will run liquidsoap with the script *1.liq*. 
   `argv` should be equal to `["./1.liq", "2.liq"]`.
1. If you want to add libraries, you can use the `-L, --lib` option which allows for multiple arguments.
   `liquidsoap -L 2.liq --lib 3.liq 1.liq 1 2` will run liquidsoap with the script *1.liq*.
   Additionally, *2.liq* and *3.liq* will be included.
   `argv` should be equal to `["1.liq", "1", "2"]`.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by Moonbase59** [comment]
Makes sense. Especially consider one might want to use LS to do some local stuff, like compute things on files, or create tests, mixes, etc.
